packages/@sfx/sayt-plugin/package.json

```diff
@@ -11,11 +14,9 @@
     "test:browser": "karma start karma.conf.js",
     "tdd:browser": "karma start karma.conf.js --no-single-run"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/groupby/sfx-logic.git",
-    "directory": "packages/@sfx/sayt-plugin"
-  },
   "author": "",
-  "license": "MIT"
+  "license": "ISC",
```

Comment:
This package, along with the other packages, are licensed under the MIT. Please revert this line.